### ISSUE TITLE
Can't select submenu item with NVDA reader on

### ISSUE THREAD

**Issue by maspychaj-sage** [Sage/carbon#4797]
### Current behaviour

When NVDA is on then there is no option to see submenu using keyboard (tab to focus main element works but enter/space/arrows doesn't open submenu).

### Expected behaviour

Submenu should be visible the same way as with disabled NVDA.

### CodeSandbox or Storybook URL

https://codesandbox.io/s/distracted-https-40him?file=/src/index.js

### JIRA Ticket (Sage Only)

SBS-27049

### Carbon Version

104.1.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by nicktitchmarsh** [comment]
@tempertemper Should we be using links in the Menu component or should we convert these to buttons?

**Comment by nicktitchmarsh** [comment]
FE-4952

**Comment by nicktitchmarsh** [comment]
An additional consideration is that some products use submenu item as a navigable link. Which is not a great accessible experience. I think we should implement Martin's recommendations but need approval from UX that all relevant designs have been changed before merging this. @harpalsingh can you advise on this?

**Comment by benwilsonsage** [comment]
Hi @tempertemper, @nicktitchmarsh , @harpalsingh
Just reviewed navigation IA in Accounting, Start, and Accounting's Accountants Nav - the only change I think we'd need to make is to make sure 'Summary' is it's own top level item in Accounting's Accountants Nav. The other overview items aren't valuable or often used, as the children cover the same content.
I'll update the global nav designs on this basis.
Happy for the above solution to go ahead.

**Comment by robinzigmond** [comment]
The above fix - changing <a> with empty <href> to a <button> - has already been implemented a few weeks ago as part of different issue/ticket, but unfortunately retesting the above reveals there are still problems specifically with NVDA on Windows. We are investigating now and will prioritise fixing this issue on our backlog.
